iml-gui/crate/src/components/date.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{
+    components::{attrs, tooltip, Placement},
+    generated::css_classes::C,
+};
+use chrono::{offset::FixedOffset, DateTime};
+use seed::{prelude::*, *};
+
+#[derive(Default)]
+pub struct Model {
+    pub(crate) basedate: Option<DateTime<FixedOffset>>,
```

Comment:
Eventually, this model can hold user's preferences on how dates should be displayed.